### ISSUE TITLE
ImportError: No module named datahub

### ISSUE THREAD

**Issue by zaydtolentino** [abhardwaj/datahub#27]
/home/ubuntu/datahub/src/apps/dbwipes/views.py:6: DeprecationWarning: the md5 module is deprecated; use hashlib instead
  import md5

Internal Server Error: /
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 103, in get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 319, in resolve
    for pattern in self.url_patterns:
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 347, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 342, in urlconf_module
    self._urlconf_module = import_module(self.urlconf_name)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/importlib.py", line 35, in import_module
    __import__(name)
  File "/home/ubuntu/datahub/src/browser/urls.py", line 171, in <module>
    url(r'^apps/dbwipes/', include('dbwipes.urls')), # dbwipes app
  File "/usr/local/lib/python2.7/dist-packages/django/conf/urls/__init__.py", line 25, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/importlib.py", line 35, in import_module
    __import__(name)
  File "/home/ubuntu/datahub/src/apps/dbwipes/urls.py", line 2, in <module>
    import views
  File "/home/ubuntu/datahub/src/apps/dbwipes/views.py", line 16, in <module>
    from service.handler import DataHubHandler
  File "/home/ubuntu/datahub/src/service/handler.py", line 7, in <module>
    from datahub import DataHub
ImportError: No module named datahub
[27/Apr/2015 03:19:12] "GET / HTTP/1.1" 500 122244

**Comment by zaydtolentino** [comment]
Hi.

Keep encountering "no module named..." errors every time I execute the command python src/manage.py syncdb.

As per instruction, I kept running pip install -r requirements.txt but still encounters missing module error.
Please assist on this.

postgres@ip-10-237-232-101:/home/ubuntu/datahub$ python src/manage.py syncdb
Traceback (most recent call last):
  File "src/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 453, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 392, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 272, in fetch_command
    klass = load_command_class(app_name, subcommand)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 77, in load_command_class
    module = import_module('%s.management.commands.%s' % (app_name, name))
  File "/usr/local/lib/python2.7/dist-packages/django/utils/importlib.py", line 35, in import_module
    __import__(name)
  File "/usr/local/lib/python2.7/dist-packages/south/management/commands/__init__.py", line 10, in <module>
    import django.template.loaders.app_directories
  File "/usr/local/lib/python2.7/dist-packages/django/template/loaders/app_directories.py", line 25, in <module>
    raise ImproperlyConfigured('ImportError %s: %s' % (app, e.args[0]))
django.core.exceptions.ImproperlyConfigured: ImportError console: No module named console

**Comment by RogerTangos** [comment]
Sorry for the delayed response. I'm working mostly on other projects at the moment, and @abhardwaj  is concentrating on a complete rewrite of datahub.

I gave this a shot, and had the same problem. This error disappeared (and a new one appeared) after I created a `/user_data` directory.

`# install and use virtualenv. seriously. it'll save you some future grief.`
`# install postgres`
`$mkdir /user_data`
`#edit settings.py to match your postgres username/password`
`$python src/manage.py syncdb`
`$python manage.py runserver`

https://github.com/abhardwaj/datahub/wiki/Getting-Started#create-user_data-directory
